src/common/graph/sieve.js

```diff
@@ -74,7 +76,7 @@ export function insertNode(current, change, result, start = 0) {
 function insertNodeIntoRange(current, index, change, result) {
   const key = change.key;
   const range = current[index];
-  const newChange = getNewer(change, range.version);
+  const newChange = getNewer(change, range);
```

Comment:
This is just for consistency with `merge`. Sometime in the future we plan to add some validation in the getNewer function to detect version inconsistencies, and this will be useful for that.